src/Community.VisualStudio.Toolkit.Shared/VS.cs

```diff
@@ -47,13 +48,25 @@ public static class VS
         public static async Task<TInterface> GetServiceAsync<TService, TInterface>() where TService : class where TInterface : class
         {
 #if VS14
-            var service = (TInterface)await AsyncServiceProvider.GlobalProvider.GetServiceAsync(typeof(TService));
+            return (TInterface)await AsyncServiceProvider.GlobalProvider.GetServiceAsync(typeof(TService));
```

Comment:
Unrelated to your change, but this is buggy because it can lead to the cast of an STA COM object on a background thread.
```suggestion
            await ThreadHelper.JoinableTaskFactory.SwitchToMainThreadAsync();
            return (TInterface)await AsyncServiceProvider.GlobalProvider.GetServiceAsync(typeof(TService));
```

The other two sections are OK because the methods that take two generic args are responsible to do the thread switch.